### ISSUE TITLE
Cannot achieve good restoration effect on RGB images

### ISSUE THREAD

**Issue by mankasto** [Koukyosyumei/AIJack#58]
Hi, thank you very much for your work. I am more interested in gradient inversion attacks, especially the work from the paper "See through Gradients: Image Batch Recovery via GradInversion". Referring to your documentation, I got good results with a smaller batch size on the MNIST dataset, but switched to RGB image datasets like cifar100, the effect was very unsatisfactory. I would like to ask if you have successfully reproduced the effect in the paper or have any suggestions for the setting of hyperparameters? I'll be grateful if you can reply as soon as possible.

**Comment by Koukyosyumei** [comment]
@mankasto 
Thank you for your interest in our project! Could you share your current model architecture, batch size, and other hyperparameters?

**Comment by Koukyosyumei** [comment]
@mankasto 
Sorry for the late response. I have prepared a notebook that shows the example of the reconstruction of CIFAR-100 with multiple batch sizes. I didn't tune hyper-parameters, so you might be able to improve them. 

https://colab.research.google.com/drive/1KShVaSQd_4uqZk9l2OQQ-imszfimawf9?usp=sharing

The original paper "See through Gradients: Image Batch Recovery via GradInversion" uses the scale parameter $\alpha_G$, which AIJack does not support. So, you may need to re-scale other parameters like TV weight. 

I guess supporting $\alpha_G$ is easy, so I would like you to create a pull request for it if you are interested.

Example (batch_size = 5)

true
![true](https://user-images.githubusercontent.com/54299088/169736273-86fcc7a2-7675-4922-bbff-5dd2d20fc810.png)

reconstructed images
![rec](https://user-images.githubusercontent.com/54299088/169736280-1ad2882a-b1b4-4cc6-9a59-f48c200d289e.png)

**Comment by Koukyosyumei** [comment]
@mankasto 

Have you solved this issue?
